psi4/driver/driver.py

```diff
@@ -82,6 +82,9 @@ def _find_derivative_type(ptype, method_name, user_dertype):
             raise ValidationError("_find_derivative_type: user_dertype should only be None or int!")
         dertype = user_dertype
 
+    if (core.get_global_option('INTEGRAL_PACKAGE') == 'ERD') and (dertype != 0):
+        raise ValidationError("INTEGRAL_PACKAGE ERD does not play nicely with gradients, so stopping.")
+
```

Comment:
Subtle errors are much worse than obvious ones. Until we have it straightened out I would very much prefer to take them all out.

You can set the catch here on the superfunctional build itself [here](https://github.com/psi4/psi4/blob/master/psi4/driver/procedures/dft_functional.py#L3065). This will exit cleanly only if ERD is set and you request a omega functional.